packages/animation-curve-block/src/components/Card.tsx

```diff
@@ -87,7 +88,13 @@ export const Card = forwardRef<HTMLDivElement, CardProps>(
                     shouldHideWrapper={replaceWithPlaceholder || !isEditing}
                     shouldHideComponent={replaceWithPlaceholder}
                     toolbarItems={[
-                        { icon: <IconArrowMove16 />, tooltip: 'Drag to move', draggableProps },
+                        {
+                            icon: <IconArrowMove16 />,
+                            tooltip: 'Drag or press ↵ to move',
+                            draggingTooltip: 'Move with ↑↓←→ and confirm with ↵',
```

Comment:
are those tooltips from the design? I'm not sure if it makes sense to keep them like this, as those actions should be imho self-explanatory. We don't do it e.g. on a delete button "press enter to delete". Wdyt?

Reply:
same for other blocks. How about to move the tooltips into the shared component?

Reply:
moved them to a constants file, so they can be used everywhere